### PR TITLE
Hashpassword suyash

### PR DESCRIPTION
**Pull request by TheHarbinger99**
hashing password to increase security done, increased password size from 50 to 500 so that saving hashed password does not cause an error. also decreased time for verification from 600 seconds to 100 seconds(need this for tests- tried a bunch of diffrent stuff very hard to even get at this point). Tests have been updated to increase coverage. 
Have pulled Dinesh's code to avoid conflict. 
Will merge this after Dinesh's PR is on develop.

### CODE REVIEW

**Comment by TheHarbinger99** [comment]
also travis will now take 3 more minutes to run, just to inform you all.

File: backend/NycBasics/models.py [hunk 1]
```diff
@@ -4,7 +4,7 @@
 class User(models.Model):
     username = models.CharField(max_length=255, null=False, unique=True)
     email = models.EmailField(max_length=255, null=False)
-    password = models.CharField(max_length=50)
+    password = models.CharField(max_length=500)
```
Comment (ds6332) [inline backend/NycBasics/models.py:7]: why do we have a 500 character limit? is it necessary?

Comment (ds6332) [reply backend/NycBasics/models.py:7]: is it because the hash has a bigger string length?

**Review by ds6332 — APPROVED** [review]
lg

File: backend/NycBasics/models.py (hunk 1 above)
Comment (TheHarbinger99) [reply backend/NycBasics/models.py:7]: yes

Comment (TheHarbinger99) [reply backend/NycBasics/models.py:7]: bigger length for hash

**Review by ds6332 — APPROVED** [review]
lg

**Review by as14680 — APPROVED** [review]
lgtm, do show a demo of what was covered in the scrum

**Review by AdnanAhsanNYCity — APPROVED** [review]
lgtm

**Pull request merged**